From d5b18e9c0986de75a905897205afa2b670b2dec9 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Mon, 28 Oct 2013 08:53:54 +0100
Subject: [PATCH] Don't check current display for extensions

Build all.
---
 Makefile.PL | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index efc21c3..200842b 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -596,9 +596,9 @@ my $build_config =
   {
     FILES =>
       "Config.pm ".
-      "utils/glversion.txt ".
-      "utils/glversion$Config{exe_ext} ".
-      "utils/glversion$Config{obj_ext}"
+      "utils/glversion.txt "
+      #"utils/glversion$Config{exe_ext} ".
+      #"utils/glversion$Config{obj_ext}"
   }
 };
 
@@ -814,6 +814,7 @@ sub get_extensions
     print "GLUT not found\n";
   }
 
+=cut
   # Platform-specific makefiles for glversion
   my $make_ver;
   if ($IS_STRAWBERRY)
@@ -876,7 +877,9 @@ sub get_extensions
      print "get_extensions: no extensions found in $glv_file\n" if $verbose;
      return '';
   }
-
+=cut
+  use Config;
+  my $gldata = { };
 
   # Parse glext_procs.h file
   return '' if (!open(GLEXT,"glext_procs.h"));
@@ -895,10 +898,12 @@ sub get_extensions
 
   # Create gl_exclude.h
   die "Unable to write to $exc_file\n" if (!open(GLEXC,">$exc_file"));
+=cut
   print GLEXC "// OpenGL Extension Exclusions - may be modified before building.\n";
   print GLEXC "//\n";
   print GLEXC "// Generated for ".$gldata->{VENDOR}.", ".$gldata->{RENDERER}."\n";
   print GLEXC "// OpenGL v".$gldata->{VERSION}.", using ";
+=cut
 
   # Fix GLUT flags based on results
   if ($gldata->{GLUT} > 0)
-- 
1.8.3.1

